fenics_concrete/material_problems/concrete_thix_mechanical.py

```diff
@@ -0,0 +1,413 @@
+import dolfin as df
+import numpy as np
+import scipy.optimize
+
+
+from fenics_concrete.material_problems.material_problem import MaterialProblem
+
+from fenics_concrete.helpers import Parameters
+from fenics_concrete.helpers import set_q
+from fenics_concrete.helpers import LocalProjector
+from fenics_concrete import experimental_setups
+
+
+import warnings
+from ffc.quadrature.deprecation import QuadratureRepresentationDeprecationWarning
+
+df.parameters["form_compiler"]["representation"] = "quadrature"
+warnings.simplefilter("ignore", QuadratureRepresentationDeprecationWarning)
+
+# copy from concrete_thermo_mechanical.py
+# change/ adapted models for modelling structural build-up
+class ConcreteThixMechanical(MaterialProblem):
+    def __init__(self, experiment=None, parameters=None, pv_name='pv_output_concrete-thix'):
+        # generate "dummy" experiment when none is passed
+        if experiment == None:
+            experiment = experimental_setups.get_experiment('MinimalCube', parameters)
+
+        super().__init__(experiment, parameters, pv_name)
+
+    def setup(self):
+        # setup initial material parameters
+        default_p = Parameters()
+        # Material parameter for concrete model with structural build-up
+        default_p['density'] = 2070  # in kg/m^3 density of fresh concrete see Wolfs et al 2018
+        default_p['u_bc'] = 0.1 # displacement on top
+
+        # temperature dependency on structural build-up not yet included
+        default_p['T'] = 22 # current ambient temperature in degree celsius
+        default_p['T_ref'] = 25  # reference ambient temperature in degree celsius
+
+        # polynomial degree
+        default_p['degree'] = 2  # default boundary setting
+
+        ### paramters for mechanics problem
+        default_p['nu'] = 0.3       # Poissons Ratio see Wolfs et al 2018
+        default_p['E_0'] = 15000    # Youngs Modulus Pa # random values!!
+        default_p['R_E'] = 15       # reflocculation rate of E modulus in Pa / s
+        default_p['A_E'] = 30       # structuration rate of E modulus in Pa / s
+        default_p['t_f'] = 300      # reflocculation time in s
```

Comment:
I assume these are the "standard" names for these parameters, corresponding to the relevant literature. Otherwise a more clear name might be nice (my model is not any better...).
Is there a reason for E_0 to be named this way? Maybe a comment of what kind of Youngs Modulus this is would be a good idea.

Reply:
E_0 is Young's modulus at t=0 ;)
I will add some more infos